pkg/util/github/github.go

```diff
@@ -125,7 +125,7 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		return errors.Wrap(err, fmt.Sprintf("Cannot download OpenShift release asset %d", assetID))
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading %s %s\n", binaryType.String(), *release.TagName)
+		fmt.Println(fmt.Sprintf("Downloading OpenShift client binary %s with version '%s'", binaryType.String(), *release.TagName))
```

Comment:
I am tend to agree with @hferentschik here to make use of general approach "Downloading OpenShift binary ...." to align with code.

Now result is : 
```
$ minishift start 
Starting local OpenShift cluster using 'kvm' hypervisor...
Downloading OpenShift binary 'oc' in version 'v1.5.0-rc.0'
 1.94 MB / 19.96 MB [==============>-------------------------------------------------------------------------------------------------------------------------------------------]   9.73% 1m10s
```